pkg/didcomm/protocol/issuecredential/states_test.go

```diff
@@ -335,18 +335,27 @@ func TestRequestReceived_CanTransitionTo(t *testing.T) {
 }
 
 func TestRequestReceived_ExecuteInbound(t *testing.T) {
-	followup, action, err := (&requestReceived{}).ExecuteInbound(&metaData{})
-	require.NoError(t, err)
-	require.Equal(t, &credentialIssued{}, followup)
-	require.NotNil(t, action)
+	t.Run("Successes", func(t *testing.T) {
+		followup, action, err := (&requestReceived{}).ExecuteInbound(&metaData{issueCredential: &IssueCredential{}})
+		require.NoError(t, err)
+		require.Equal(t, &credentialIssued{}, followup)
+		require.NotNil(t, action)
 
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
 
-	messenger := serviceMocks.NewMockMessenger(ctrl)
-	messenger.EXPECT().ReplyTo(gomock.Any(), gomock.Any())
+		messenger := serviceMocks.NewMockMessenger(ctrl)
+		messenger.EXPECT().ReplyTo(gomock.Any(), gomock.Any())
 
-	require.NoError(t, action(messenger))
+		require.NoError(t, action(messenger))
+	})
+
+	t.Run("IssueCredential is absent", func(t *testing.T) {
+		followup, action, err := (&requestReceived{}).ExecuteInbound(&metaData{})
+		require.Contains(t, fmt.Sprintf("%v", err), "issue credential was not provided")
```

Comment:
replace `fmt.Sprintf("%v", err)` with `err.Error()`

Reply:
This is not a good idea. `err.Error()` will cause a panic when `err` is `nil`. The best way to check it is to use `require.EqualError`. But `EqualError` requires full error string. Which causes a linter issue (lll e.g line is too long). 
So, I figured out that the best way is to use `Contains`. 
I can try to replace `Contains` with `EqualError`. But it will work only for errors with small length. Otherwise, I need to disable the linter.

Reply:
this is an expected error test case, so err should be not nil, if it is, let the test panic, that's fine